cli/src/alluxio.org/cli/env/command.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/log"
+)
+
+type Command interface {
+	Base() *BaseCommand
+	Run([]string) error
+	ToCommand() *cobra.Command
+}
+
+type BaseCommand struct {
+	CommandName   string
+	JavaClassName string
```

Comment:
I like this: it keeps the go cli command right next to the Java class name, it's clever.